src/_locales/ru/messages.json

```diff
@@ -29,23 +29,23 @@
     "html_about_github_title": { "message": "GitHub" },
     "html_about_twitter_title": { "message": "Twitter" },
     "html_about_report_issues_prefix": { "message": "Связаться с нами или сообщить о баге можно на" },
-    "html_about_report_issues_suffix": { "message": ". Пожалуйста. убедитесь что ваша идея или предложение не опубликованы ранее. Спасибо!" },
+    "html_about_report_issues_suffix": { "message": ". Пожалуйста, убедитесь, что Ваша идея или предложение не были опубликованы ранее. Спасибо!" },
     "html_about_debugging_prefix": { "message": "Чтобы включить ведение журнала или установить флаги отладки, перейдите на " },
     "html_about_debugging_suffix": { "message": "страницу отладки." },
     "html_about_the_github_issues_page": { "message": "странице предложений GitHub" },
     "html_about_html2canvas": { "message": "html2canvas" },
     "html_about_html2canvas_prefix": { "message": "Это расширение использует" },
     "html_about_html2canvas_suffix": { "message": "библиотеку Niklas von Hertzen." },
     "html_about_indexedDb": { "message": "db.js" },
-    "html_about_indexedDb_prefix": { "message": "Также используется обёртка indexedDb " },
+    "html_about_indexedDb_prefix": { "message": "Также используется indexedDb враппер " },
     "html_about_indexedDb_suffix": { "message": "от Aaron Powell." },
     "html_about_browserstack": { "message": "browserstack" },
     "html_about_browserstack_prefix": { "message": "Отдельное спасибо" },
-    "html_about_browserstack_suffix": { "message": "за бесплатные средства отладки chrome." },
+    "html_about_browserstack_suffix": { "message": "за бесплатные средства отладки Chrome." },
     "html_about_feeling_generous": { "message": "Вы можете поддержать The Great Suspender, отправив пожертвование:" },
     "html_about_already_donated": { "message": "Уже пожертвовали?" },
     "html_about_thanks_for_your_donation": { "message": "Большое спасибо за пожертвование. Наш кот теперь доволен." },
-    "html_about_want_to_donate_again": { "message": "Отправите еще пожертвование?" },
+    "html_about_want_to_donate_again": { "message": "Хотите пожертвовать еще?" },
```

Comment:
`ещё` (there is 33th letter of the Russian alphabet).

Reply:
```suggestion
    "html_about_want_to_donate_again": { "message": "Хотите пожертвовать ещё?" },
```

Reply:
here I may agree especially considering "ещё" being at the end of the sentence

Reply:
In (semi-) official docs the letter “ё” is not used, it is not even in all rus fonts.
It can also be a problem if someone changes the standard font in chrome.
so to me, like all Russian, it does not matter, we are used to reading without ё

Reply:
> In (semi-) official docs the letter “ё” is not used

Any proofs for this?

> it is not even in all rus fonts

Font can't be "rus", it can support Cyrillic less or more. Support Cyrillic without `ё` is marker of bad font.

> It can also be a problem if someone changes the standard font in chrome.

It's a problem of font or person who chose such font.

> we are used to reading without ё

Please, don't claim for all Russian speakers. I like `ё`. This is official letter in the Russian alphabet. `ё` and `е` are different letters. Like `все` and `всё` are different words (there is such example in your changes). Please, use correct letters in words.

Reply:
@AlexWayfer idk but u r being too grammar nazi here imo, even if generally I'm pro "ё", font thing is real

Reply:
> @AlexWayfer idk but u r being too grammar nazi here imo

Yes. I want as correct texts as possible.

> font thing is real

And `ё` is real, and the difference between `все` and `всё` is real. You're trying to make a work-around for incorrect (broken, incomplete) fonts. I suggest you to create an issue for specific (bad) fonts. Or just not to use them.

Reply:
@AlexWayfer 
если вы русский, то вы прекрасно знаете это, если нет, тогда почему вы столь недоверчивы?
![image](https://user-images.githubusercontent.com/23438485/52355069-acfd4b80-2a42-11e9-8785-0f1d350da881.png)
можете скачать любую русскоязычную инструкцию или книгу и проверить сами

Reply:
> если вы русский

Знание языка не связано с национальностью, хотя и коррелирует. Да, я свободно разговариваю на русском.

> то вы прекрасно знаете это

Ошибочное утверждение, основанное на собственном опыте.

> можете скачать любую русскоязычную инструкцию или книгу и проверить сами

Мне не нужно скачивать, у меня есть книги. Например, Толковый словарь Ожегова. В нём используется буква "ё". Не подходит?

Я всё ещё против замены одной буквы на другую. Хоть они и схожи, но не зря они разные, и до сих пор существуют вместе.

Reply:
Ok well, I get ur point, I would let @MonsterNya to decide here as he started talking about fonts and I just know it's a real thing as well as I see this "е" or "ё" too

Reply:
> he started talking about fonts and I just know it's a real thing

I have not seen this. I had problems with Cyrillic, but not with `ё`.

Reply:
I do not argue with the letter ё in terms of grammar. but from the point of view of the ASCII code of the letters ё Ё are in the section of special symbols.
when I was working with fonts, I found that not all fonts that have Cyrillic support are declared to have the characters ё Ё and even №. So the use of letters ё is due to the technical side, but I do not know how relevant it is in our case.

Reply:
> but from the point of view of the ASCII code of the letters ё Ё are in the section of special symbols.

ASCII contains 128 characters (7 bit), and there is no Cyrillic at all.

> when I was working with fonts, I found that not all fonts that have Cyrillic support are declared to have the characters ё Ё and even №.

I don't think that these fonts are good for people needing Cyrillic characters.

For example, [Google Fonts is having 103 of 915 fonts now, supporting Cyrillic](https://fonts.google.com/?subset=cyrillic). If several of them don't support `ё` — OK, I'll reconsider.

Reply:
http://foxtools.ru/ASCII#1251

Reply:
> http://foxtools.ru/ASCII#1251

I don't trust this site.

Please, take a look at https://en.wikipedia.org/wiki/ASCII or http://www.asciitable.com/ or https://ascii.cl/

Reply:
Вы мне предлагаете найти в US-ASCII русские символы?
https://ru.wikipedia.org/wiki/ASCII#Национальные_варианты_ASCII
Ищите в гугле Extended ASCII/Unicode. Я не знаю, специально ли вы придираетесь. Открыв какой-нибудь шрифт через редактор вы можете сами убедится, где и как расположены ё Ё.
И есть ли они в шрифтах "с кириллицей", которые вы можете найти на подобных англоязычных сайтах.

Reply:
> Вы мне предлагаете найти в US-ASCII русские символы?

Вы сослались на ASCII. ASCII состоит из 7 бит. Есть "расширенная". ASCII — её половина:

> ASCII стала восприниматься как половина 8-битной кодировки, а «расширенной ASCII» называли ASCII с задействованным 8-м битом (например, КОИ-8).

---

> Ищите в гугле Extended ASCII/Unicode.

Unicode — вообще далеко от ASCII по объёму:

> Unicode 11.0, contains a repertoire of 137,439 characters covering 146 modern and historic scripts, as well as multiple symbol sets and emoji.

Вместо 128 или 258 символов.

---

> Открыв какой-нибудь шрифт через редактор вы можете сами убедится, где и как расположены ё Ё.

Мне это не нужно. И не удивлюсь, если зависит от редактора. Я верю, что `ё` располагается не между `е` и `ж` в некоторых вещах, на практике сталкивался. Однако, мы сейчас обсуждаем наличие реализации, а не порядок. Реализация должна быть, так как это часть кириллицы. И мир от ASCII (в любой реализации) переходит к Unicode (включая Emoji).

---

> И есть ли они в шрифтах "с кириллицей", которые вы можете найти на подобных англоязычных сайтах.

В каких шрифтах? Каких сайтах? Подобных чему? Я не понимаю, мало конкретики.

Reply:
Вот именно, мы обсуждаем реализацию, а не полноту ASCII.
У меня нет статистики по шрифтам, но я знаю, что это проблема. На сколько её можно выразить в процентах? Я не знаю. 
На этом я предлагаю закончить переливать из пустого в порожнее.

Reply:
> На этом я предлагаю закончить переливать из пустого в порожнее.

И заниматься переводом, а не проблемами полноты шрифтов без практических отзывов (от пользователей расширения) и чисел.

`ё` — это не какой-то новый спец. символ, появившийся в стандарте Unicode года 2 назад, как символ рубля (`₽`), например. И то который уже достаточно распространён. 

Во многих ОС также есть система fallback, когда при отсутствии символа в шрифте используется "запасной" шрифт.